bot/actions/slack_incident_actions.rb

```diff
@@ -58,6 +109,11 @@ def notify_channel!(channel, incident_channel_id, title, priority)
                                   text: notification_text)
   end
 
+  def notify_channel_of_update!(channel_id)
```

Comment:
We could probably at some point come up with a `SlackChannel` object that offered methods like `send_message` etc?

Reply:
Will add a new issue to cover this!